src/main/java/core/basesyntax/FigureArea.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface FigureArea {
+    default double getArea() {
+        return 0;
+    }
```

Comment:
```suggestion
   double getArea();
```
You do not need to add default implementation